Feat: Add meeting minutes 2026-01-28 (#1461)
oasis-tcs/csaf: meeting_minutes/2026/2026-01-28.md

![image](https://user-images.githubusercontent.com/1690898/139102180-5c1e2583-14f1-4f58-ab2b-9e3807ed529c.png)

# Common Security Advisory Framework (CSAF) Technical Committee Working Meeting

- Meeting Date: January 28, 2026
- Time: 18:00 UTC (19:00 CET, 13:00 EST, 10:00 PST)

## Call to Order and Welcome

- Meeting called to order @ 18:07 UTC.

## Roll call

All participants recorded their attendance on the OQC system.

## Participants

> Quorum requires participation of 5 or more of the 9 (10 - 1) voting members (including the officers).

Reviewer: Please remind me: Why do we state with `10 - 1` voting members? OQC states there should be 10 and I don't see a LoA in my notes... 
Otherwise, we would have to state:

```suggestion
Quorum requires participation of 6 or more of the 10 voting members (including the officers).
```

| Given Name | Family Name | Affiliation                                                 | Role             | Present |
|:-----------|:------------|:------------------------------------------------------------|:-----------------|:--------|

> | Celia      | Savidge     | Dell                                                        | Observer         | Yes     |

Reviewer: As Celia was still an observer at that point, we need to remove her from the member list.
```suggestion
```

| Christian  | Banse       | Fraunhofer-Gesellschaft e.V. - Fraunhofer AISEC             | Voting Member    | Yes     |
| Christoph  | Plutte      | Ericsson AB                                                 | Voting Member    | Yes     |

> | Dejan      | Bosanac     | Red Hat                                                     | Observer         | Yes     |

Reviewer: Same here
```suggestion
```

| Dina       | Truxius     | Federal Office for Information Security (BSI)               | Member           | Yes     |
| Feng       | Cao         | Oracle                                                      | Voting Member    | Yes     |

> | JD         | Stefaniak   | Dell                                                        | Member           | No      |

Reviewer: He was still a voting member by then - lost his voting rights after the meeting.
```suggestion
| JD         | Stefaniak   | Dell                                                        | Voting Member    | No      |
```

| Jane       | Ginn        | Cyber Threat Intelligence Network, Inc.                     | Member           | No      |
| Jessie     | Vaught      | Red Hat                                                     | Voting Member    | Yes     |
| Johannes   | Lorenz      | Siemens AG                                                  | Member           | No      |

> | Justin     | Murphy      | DHS Cybersecurity and Infrastructure Security Agency (CISA) | Voting Member    | Yes     |

Reviewer: He lost voting rights before, but should be listed here as Co-Chair.
```suggestion
| Justin     | Murphy      | DHS Cybersecurity and Infrastructure Security Agency (CISA) | Co-Chair         | Yes     |
```

| Michael    | Reeder      | Dell                                                        | Member           | Yes     |

> | Michael    | Schueler    | Cisco Systems                                               | Voting Member    | Yes     |

Reviewer: He just gained voting rights after the meeting.
```suggestion
| Michael    | Schueler    | Cisco Systems                                               | Member           | Yes     |
```

| Sonny      | van Lingen  | Huawei Technologies Co., Ltd.                               | Voting Member    | Yes     |
| Stefan     | Hagen       | Individual                                                  | Co-Chair         | Yes     |
| Thomas     | Proell      | Siemens AG                                                  | Member           | No      |
| Thomas     | Schaffer    | Cisco Systems                                               | Member           | Yes     |
| Thomas     | Schmidt     | Federal Office for Information Security (BSI)               | Voting Member    | Yes     |
| Tobias     | Limmer      | Siemens AG                                                  | Member           | Yes     |
| Vijay      | Sarvepalli  | SEI                                                         | Member           | No      |
| Vivek      | Nair        | Microsoft Corporation                                       | Member           | Yes     |

### Observers present

Note: Observers of this committee that are ready to become Members should follow the specific instructions displayed the OASIS Open Notices tab.

> 

Reviewer: ```suggestion

- Celia Savidge (Dell)
- Dejan Bosonac (Red Hat)

```

Author: Thanks! Addressed.

### Quorum

- Quorum was **reached**.

## Agenda

- Roll Call
- Updates or news to the TC
- Review all open GitHub issues and pull requests requiring TC discussion.
- Next steps
- Adjourn

## Identification of TC voting members (Secretary)

### Prospective voting members attending their first meeting

### Members attaining voting rights at the end of this meeting

> 

Reviewer: ```suggestion

- Justin Murphy
- Michael Schueler

```

Author: Thanks! Addressed.

### Members losing voting rights if they have not joined this meeting by the time it ends

> 

Reviewer: ```suggestion

- JD Stefaniak
- Omar Santos

```

Author: Thanks! Addressed.

### Members who previously lost voting rights who are attending this meeting

> 

Reviewer: ```suggestion

- Dina Truxius
- Michael Reeder

```

Author: Thanks! Addressed.

### Members who have declared a leave of absence

## Meeting Notes

### Updates and Introductions

- Celia Savidge introduced herself as the new program manager for the CSAF initiative at Dell.
- Dejan Bosanac joined the meeting as an observer to learn about ongoing projects and contribute in the future.

### Motions and Standing Rules

> - The group discussed motions related to CSAF 2.1, including superseding pending motions.

Reviewer: We need to be more specific here: issues number, What was the issue, who set the motion, who second it, where there discussions / objections / concerns? Did the motion pass?

Author: Thanks! Addressed.

> - The TC approved a standing rule allowing editors to add items to known issues without requiring an extra motion.

Reviewer: For approving a standing rule, a Full Majority Vote is held. The meeting minutes need to reflect that.

Author: Thanks! Addressed.

| Thomas     | Schaffer    | Cisco Systems                                               | Member           | Yes     |
| Thomas     | Schmidt     | Federal Office for Information Security (BSI)               | Voting Member    | Yes     |
| Tobias     | Limmer      | Siemens AG                                                  | Member           | Yes     |
| Vijay      | Sarvepalli  | SEI                                                         | Member           | No      |
| Vivek      | Nair        | Microsoft Corporation                                       | Member           | Yes     |

### Observers present

- Celia Savidge (Dell)
- Dejan Bosonac (Red Hat)

Note: Observers of this committee that are ready to become Members should follow the specific instructions displayed the OASIS Open Notices tab.

### Quorum

- Quorum was **reached**.

## Agenda

- Roll Call
- Updates or news to the TC
- Review all open GitHub issues and pull requests requiring TC discussion.
- Next steps
- Adjourn

## Identification of TC voting members (Secretary)

### Prospective voting members attending their first meeting

### Members attaining voting rights at the end of this meeting

> ### Members losing voting rights if they have not joined this meeting by the time it ends
> 
> - Justin Murphy
> - Michael Schueler
> 
> ### Members who previously lost voting rights who are attending this meeting
> 
> - JD Stefaniak
> - Omar Santos
> 
> ### Members who have declared a leave of absence
> 
> - Dina Truxius
> - Michael Reeder

Reviewer: I guess there was a heading confusion...

```suggestion
### Members attaining voting rights at the end of this meeting

- Justin Murphy
- Michael Schueler

### Members losing voting rights if they have not joined this meeting by the time it ends

- JD Stefaniak
- Omar Santos

### Members who previously lost voting rights who are attending this meeting

- Dina Truxius
- Michael Reeder
- 

### Members who have declared a leave of absence

```

Author: Comment addressed in the recent changes. Thank You!

- Celia Savidge (Dell) introduced herself as the new program manager for the CSAF initiative at Dell, taking over from Michael Reeder, who is transitioning to other projects. Justin Murphy offered to check the member list on the Higher Logic platform to ensure Celia is added.
- Dejan Bosanac (Red Hat) introduced himself as a new observer; he presented Trustify at the CSAF Community Days in October and joined to learn about ongoing work and contribute in the future. Justin Murphy welcomed him and offered help with OASIS administrator access. Stefan Hagen confirmed Dejan is already on the roster and registration is in progress.

### GitHub Issues & Pull Requests

#### Super-motion to supersede pending motions (PRs [#1274](https://github.com/oasis-tcs/csaf/pull/1274), [#1278](https://github.com/oasis-tcs/csaf/pull/1278), [#1281](https://github.com/oasis-tcs/csaf/pull/1281), [#1282](https://github.com/oasis-tcs/csaf/pull/1282))

- Thomas Schmidt opened with a super-motion to supersede all pending motions except the motion for PR #1221 (where there was still discussion to be had).

> - A motion was presented by Thomas Schmidt to supersede the pending motions for PRs #1274, #1278, #1281 and #1282 with the content of accepting those pull requests into CSAF 2.1. No objections, discussions or concerns were raised. The motion carries.

Reviewer: ```suggestion
- A motion was presented by Thomas Schmidt to supersede the pending motions for PRs #1274, #1278, #1281 and #1282 with the content of accepting those pull requests into CSAF 2.1. Christian Banse seconded. No objections, discussions or concerns were raised. The motion carries.
```

Author: Comment addressed in the recent changes. Thank You!
